settings/serverSettings.ts

```diff
@@ -1,94 +1,100 @@
 // This is where server-side only, potentially sensitive settings enter from the environment
 // DO NOT store sensitive strings in this file itself, as it is checked in to git!
 
-import fs from "fs-extra"
 import path from "path"
+import dotenv from "dotenv"
+dotenv.config()
 
-export const BASE_DIR = path.resolve(__dirname, "../..")
-const absoluteSettingsPath = path.resolve(BASE_DIR, "serverSettings.json")
-const localOverrides: any = fs.existsSync(absoluteSettingsPath)
-    ? fs.readJsonSync(absoluteSettingsPath)
-    : {}
+import * as clientSettings from "./clientSettings"
+import { parseIntOrUndefined } from "../clientUtils/Util"
 
-export const ADMIN_SERVER_PORT = localOverrides.ADMIN_SERVER_PORT ?? 3030
-export const ADMIN_SERVER_HOST = localOverrides.ADMIN_SERVER_HOST ?? "localhost"
-export const BAKED_BASE_URL =
-    localOverrides.BAKED_BASE_URL ??
-    `http://${ADMIN_SERVER_HOST}:${ADMIN_SERVER_PORT}`
+const serverSettings = process.env ?? {}
 
-export const BAKED_GRAPHER_URL =
-    localOverrides.BAKED_GRAPHER_URL ?? `${BAKED_BASE_URL}/grapher`
+export const BASE_DIR: string = path.resolve(__dirname, "../..")
+export const ENV: "development" | "production" = clientSettings.ENV
 
-export const ADMIN_BASE_URL =
-    localOverrides.ADMIN_BASE_URL ??
-    `http://${ADMIN_SERVER_HOST}:${ADMIN_SERVER_PORT}`
+export const ADMIN_SERVER_PORT: number = clientSettings.ADMIN_SERVER_PORT
+export const ADMIN_SERVER_HOST: string = clientSettings.ADMIN_SERVER_HOST
+export const BAKED_BASE_URL: string = clientSettings.BAKED_BASE_URL
 
-export const WORDPRESS_URL =
-    localOverrides.WORDPRESS_URL ?? "https://owid.cloud"
+export const ADMIN_BASE_URL: string = clientSettings.ADMIN_BASE_URL
+export const WORDPRESS_URL: string = clientSettings.WORDPRESS_URL
 
-export const ALGOLIA_ID = localOverrides.ALGOLIA_ID ?? ""
+export const BAKED_GRAPHER_URL: string =
+    serverSettings.BAKED_GRAPHER_URL ?? `${BAKED_BASE_URL}/grapher`
 
-export const RECAPTCHA_SITE_KEY =
-    localOverrides.RECAPTCHA_SITE_KEY ??
-    "6LcJl5YUAAAAAATQ6F4vl9dAWRZeKPBm15MAZj4Q"
+export const OPTIMIZE_SVG_EXPORTS: boolean =
+    serverSettings.OPTIMIZE_SVG_EXPORTS === "true" ?? false
```

Comment:
I feel like this one has stood the test of time and we should always have it enabled by default? (Not a change for this PR, but just wondering)